package.json

```diff
@@ -31,7 +31,6 @@
     "cross-spawn": "^6.0.5",
     "css-loader": "^0.28.11",
     "file-loader": "^1.1.11",
-    "gitdiff-parser": "^0.0.8",
     "html-webpack-plugin": "^3.2.0",
     "js-yaml": "^3.11.0",
```

Comment:
I don't think this is needed any more 😃.

Reply:
@mRcfps 

I already deleted it !

![image](https://user-images.githubusercontent.com/26423749/42420075-e9967b0e-82f2-11e8-8cfd-07f802f1ca76.png)

Reply:
Please note that I'm commenting on the line with js-yaml.

Reply:
😅

Reply:
Now, the tests files still need `js-yaml`. But in future, I'll delete it 😃.

![image](https://user-images.githubusercontent.com/26423749/42420206-d6f1114c-82f4-11e8-85fc-35194732c003.png)

Reply:
Okaaaaaay.